packages/web-client-internal/src/config-storage.shim.js

```diff
@@ -0,0 +1,12 @@
+/**!
+ *
+ * Copyright (c) 2015 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import LocalForageStoreAdapter from '@ciscospark/storage-adapter-local-forage';
+import LocalStorageStoreAdapter from '@ciscospark/storage-adapter-local-storage';
+
+export default {
```

Comment:
No need for separate shim and non-shim versions. this package is for web client consumption, so it doesn't need node support.